tests/test_sticker.py

```diff
@@ -139,6 +144,9 @@ async def test_send_all_args(self, bot, chat_id, sticker_file, sticker):
         assert message.sticker.is_animated == sticker.is_animated
         assert message.sticker.is_video == sticker.is_video
         assert message.sticker.file_size == sticker.file_size
+        assert message.sticker.type == sticker.type
+        assert message.sticker.is_premium == sticker.is_premium
+        assert message.has_protected_content
```

Comment:
relation? We should probably remove protect_content = True above
```suggestion
```

Reply:
but the test here is for passing all args to a function actually leads to all args being submitted to telegram? So that one is important?

Reply:
hmm good point